src/tests/JIT/Regression/JitBlue/Runtime_34587/Runtime_34587.cs

```diff
@@ -616,6 +623,18 @@ static bool ValidatePopcnt()
             return succeeded;
         }
 
+        static bool ValidateSha()
+        {
+            bool succeeded = true;
+
+            if (X86Sha.IsSupported)
+            {
+                succeeded &= X86Sha.IsSupported;
+            }
```

Comment:
This will need to be updated to roughly mirror `ValidateAes` where it also checks for `Sse2`